### DISCUSSION TITLE
[my first touch] MINIO domain Configuration on Ubuntu

### DISCUSSION THREAD

**Discussion by Uominiup** [minio/minio#17344] [Q&A] [answered]
Hello guys, 

This is my first testing of MINIO. My goal is it that I can use it as storage for VEEAM.

I already installed MINIO on Ubuntu and it is/was running. 

After I installed the SSL Certificate and activated the reverse proxy, the browser redirected me after typing the url “https://s3.contos.com” to “https://172.40.40.16:9000” (IP is an example) 

So how do I make it work that the MINIO uses the domain not the Ip? 

Regards 🙋‍♂️

**Comment by Uominiup** [comment]
-

**Comment by klauspost** [comment, marked as answer]
[MINIO_SERVER_URL="https://s3.contos.com"](https://min.io/docs/minio/linux/reference/minio-server/minio-server.html#envvar.MINIO_SERVER_URL)
